### PR TITLE
Update Dockerfile

### PR DESCRIPTION
**Pull request by jibla**
This is the fix for issue #2339

### CODE REVIEW

File: api/Dockerfile [hunk 1]
```diff
@@ -8,6 +8,9 @@ ENV APP_ENV=prod
 
 WORKDIR /srv/app
 
+# install linux headers package
+RUN apk add --update linux-headers
```
Comment (dunglas) [inline api/Dockerfile:12]: Can't this package added to the next RUN instruction instead?

**Review by dunglas — CHANGES_REQUESTED** [review]

File: api/Dockerfile (hunk 1 above)
Comment (jibla) [reply api/Dockerfile:12]: Sure, it can be, will move there.

Comment (jibla) [reply api/Dockerfile]: Done.

**Review by MWDylanD — APPROVED** [review]

**Review by dunglas — CHANGES_REQUESTED** [review]
Thanks! Could you preserve the original indentation please?

**Comment by jibla** [comment]
@dunglas Fixed.

File: api/Dockerfile [hunk 2]
```diff
@@ -10,12 +10,13 @@ WORKDIR /srv/app
 
 # persistent / runtime deps
 RUN apk add --no-cache \
-		acl \
-		fcgi \
-		file \
-		gettext \
-		git \
-	;
+        linux-headers \
```
Comment (RubenKruiswijk) [inline api/Dockerfile:13]:
```suggestion
		linux-headers \
```

File: api/Dockerfile [hunk 3]
```diff
@@ -10,12 +10,13 @@ WORKDIR /srv/app
...
+        acl \
```
Comment (RubenKruiswijk) [inline api/Dockerfile:14]:
```suggestion
		acl \
```

File: api/Dockerfile [hunk 4]
```diff
@@ -10,12 +10,13 @@ WORKDIR /srv/app
...
+        fcgi \
```
Comment (RubenKruiswijk) [inline api/Dockerfile:15]:
```suggestion
		fcgi \
```

File: api/Dockerfile [hunk 5]
```diff
@@ -10,12 +10,13 @@ WORKDIR /srv/app
...
+        file \
```
Comment (RubenKruiswijk) [inline api/Dockerfile:16]:
```suggestion
		file \
```

File: api/Dockerfile [hunk 6]
```diff
@@ -10,12 +10,13 @@ WORKDIR /srv/app
...
+        gettext \
```
Comment (RubenKruiswijk) [inline api/Dockerfile:17]:
```suggestion
		gettext \
```

File: api/Dockerfile [hunk 7]
```diff
@@ -10,12 +10,13 @@ WORKDIR /srv/app
...
+        git \
```
Comment (RubenKruiswijk) [inline api/Dockerfile:18]:
```suggestion
		git \
```

File: api/Dockerfile [hunk 8]
```diff
@@ -10,12 +10,13 @@ WORKDIR /srv/app
...
+    ;
```
Comment (RubenKruiswijk) [inline api/Dockerfile:19]:
```suggestion
	;
```

**Review by RubenKruiswijk — COMMENTED** [review]
I believe this is the original indentation @dunglas was looking for. :-)

**Comment by jibla** [comment]
@RubenKruiswijk applied all your suggestions. Thanks.

**Comment by dunglas** [comment]
Hi @jibla. Thanks for your work on this. I (obviously) had the same issue while working on #2344, and it looks like this package is only necessary for XDebug, so it could be added only to the dev image: https://github.com/api-platform/api-platform/pull/2344/files#diff-21ee93e31c9cec7a5f33b680622da377c451b62b6c44eac6d9550eade41beb47R116

**Comment by dunglas** [comment]
Included in #2344. Thanks again @jibla!

**Pull request closed without merging**